How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md

```diff
@@ -43,7 +43,7 @@ PRs must meet the following requirements:
 8. Code should pass all tests and linting
 9. Code should be of general high quality and a needed change or improvement
 
-If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines.  New Contributors may be referred to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) Chat room.  At at the Mod's discretion the issue may be closed.
+If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines. New Contributors may be referred to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) Chat room and the issue may also be closed at a Mod's discretion.
 

```

Comment:
I deliberately broke those into two sentences, because the actions are not directly related.  You may refer them to help chat if you think it will be helpful.  You may *also* choose to close the PR (not issue, so that's an error) if you believe it is not repairable.  For example, if they have filed it against master or from their own staging branch.

You don't have to do one or the other or both.